packages/happo-target-firefox/src/getUrlsFromSrcset.js

```diff
@@ -0,0 +1,10 @@
+const SRCSET_ITEM = /([^\s]+)(\s+[0-9.]+[wx])?(,?\s*)/g;
+
+export default function getUrlsFromSrcset(value) {
+  const result = [];
+  let match;
+  while (match = SRCSET_ITEM.exec(value)) { // eslint-disable-line no-cond-assign
```

Comment:
You can use https://npmjs.com/string.prototype.matchall for this :-D

Reply:
Thanks! I'll make that change after I publish.